### PR TITLE
Intl.Pluralrules

### PR DESCRIPTION
**Pull request by zbraniecki**
Ok, let's start the PR.

### CODE REVIEW

**Comment by zbraniecki** [comment]
It seems to cover the whole spec, but I'm not sure about how the make-plurals are supposed to work, as when I try to just launch it, it errors with:

```
/home/zbraniecki/projects/intl/Intl.js/node_modules/make-plural/es6/plurals.js:22
export default {
^^^^^^
SyntaxError: Unexpected token export
    at Object.exports.runInThisContext (vm.js:78:16)
    at Module._compile (module.js:543:28)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/zbraniecki/projects/intl/Intl.js/lib/core.js:5:31)
    at Module._compile (module.js:571:32)
```

@eemeli recommendation ( https://github.com/zbraniecki/Intl.js/pull/1#issuecomment-269814313 ) is:

```
babel-node --presets babel-preset-es2015 --ignore regenarator-transform index.js
```

which I'm not sure if it'll fly with the whole Intl.js requiring it to run.

**Comment by zbraniecki** [comment]
@caridy - but except of this error, I think the PR is ready for review.

**Comment by caridy** [comment]
Any update on this @zbraniecki ?

**Comment by zbraniecki** [comment]
So, the PR is ready, but there's a bug on travis that I don't understand, and when I'm trying to manually test using `node ./index.js` I get a different error, and @eemeli is suggesting a spell to solve it that doesn't work.

It seems like both errors are related to babel and I'm not familiar with babel at all, so I don't know how to fix either of them.

Need help.

**Comment by eemeli** [comment]
Hopefully fixed with zbraniecki/Intl.js#2.

**Comment by zbraniecki** [comment]
The travis is fixed with https://github.com/zbraniecki/Intl.js/pull/2 thank you @eemeli !

So it seems like this PR should be ready to be merged

**Comment by zbraniecki** [comment]
I applied availableLocales from make-plurals. I have no idea why travis failed now...

**Comment by zbraniecki** [comment]
@caridy ?

**Comment by shouze** [comment]
Would appreciate to get this merged, how could I help?

**Comment by zbraniecki** [comment]
It's ready to land as far as I can tell. Pending @caridy to calm travis down and land it.

**Comment by shouze** [comment]
ping @caridy @andyearnshaw what can we do to see this issue merged soon? Should @zbraniecki be granted to merge PR on this repo?

**Comment by MikaFima** [comment]
Hi, this feature is a real need by there. Can we hope a merge soon ?

File: src/11.numberformat.js [hunk 1]
```diff
@@ -66,12 +66,64 @@ defineProperty(Intl.NumberFormat, 'prototype', {
...
+    intlObj['[[minimumIntegerDigits]]'] = mnid;
+
+    // 10. Set intlObj.[[minimumFractionDigits]] to mnfd.
+    intlObj['[[minimumFractionDigits]]'] = mnfd;
+
+    // 11. Set intlObj.[[maximumFractionDigits]] to mxfd.
+    intlObj['[[maximumFractionDigits]]'] = mxfd;
+
+    // 12. If mnsd is not undefined or mxsd is not undefined, then
+    if (mnsd !== undefined || mxsd !== undefined) {
```
Comment (caridy) [inline src/11.numberformat.js]: @zbraniecki we changed this one a little bit I think, can you update the implementation?

**Comment by shouze** [comment]
Thank you so much @caridy! Do you plan to make a release too?

Would be able to upgrade https://github.com/Financial-Times/polyfill-service/issues/1097 this way.

**Comment by caridy** [comment]
@shouze unfortunately I can't do it this week, but if there is anyone willing to help with the release, I can help :)

**Comment by shouze** [comment]
ok so @caridy I can help what needs to be done to publish a 1.2.6 except publishing to npm repo?

It's been quite a long time that 1.2.6 has been prepared (https://github.com/andyearnshaw/Intl.js/commit/af235a652125f5f0d9d960707337340bb4067134) in fact so it should be released ASAP no?

Please tell me in any way I can help, can be your hands to make things happen! 👼

**Comment by caridy** [comment]
@shouze let's start with https://github.com/andyearnshaw/Intl.js/blob/master/CONTRIBUTING.md#release-checklist

**Comment by aurambaj** [comment]
Hi, any plan to release this soon?

**Pull request merged**